packages/shared/lib/core/wallet/actions/trySendAmount.ts

```diff
@@ -0,0 +1,30 @@
+import { localize } from '@core/i18n'
+import { isSoftwareProfile } from '@core/profile'
+import { showAppNotification } from '@lib/notifications'
+import { checkStronghold } from '@lib/stronghold'
+import { isTransferring } from '@lib/wallet'
+import { get } from 'svelte/store'
+import { sendAmount } from './sendAmount'
+
+export async function trySendAmount(receipientAddress: string, amount: number): Promise<void> {
+    const _send = async () => {
+        isTransferring.set(true)
+        try {
+            const transactionReceipts = await sendAmount(receipientAddress, amount)
+            isTransferring.set(false)
+        } catch (err) {
+            console.error(err)
```

Comment:
Do we accept `console.log`s on dev?

Reply:
no console logs, just console.error. We still need to agree on the error handling app wide.